apps/amo/urlresolvers.py

```diff
@@ -163,6 +163,10 @@ def fix(self, path):
         path = path.lstrip('/')
         url_parts = [self.request.META['SCRIPT_NAME']]
 
+        if path.startswith(settings.SUPPORTED_NONAPPS_NONLOCALES_DEEP):
```

Comment:
`startswith` can accept a string or an iterable, so it works either way, i've made at an iterable to be consistent.

```
>>> 'api/v3'.startswith(('api/v2', 'api/v4',))
False
>>> 'api/v3'.startswith(('api/v2', 'api/v3',))
True
>>> 'api/v3'.startswith('api/v3')
True
```

The line you mentioned uses `partition` to split the line and check the first value, this is effectively something like `^blocklist(?:/)` or whatever, I could change everything to be regex's and so on. But because `api/v3` contains a `/` it won't work unless we do `api` but then affects all the APIs on AMO.

This felt like the least invasive change on something we'll be deleting.